src/saving/InProgressLoad.cs

```diff
@@ -153,6 +155,7 @@ private void Step()
                 if (success)
                 {
                     LoadingScreen.Instance.Hide();
+                    isLoading = false;
```

Comment:
This should be set to false even if this fails. I'd put this line just before the return on line 169.